### PR TITLE
DevKit updates for 1.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 .php_cs.cache
 composer.lock
 phpunit.xml
+/.phpunit.result.cache
```
Comment (OskarStark) [inline .gitignore]: Shall we add this to dev kit?

Comment (greg0ire) [reply .gitignore]: The whole gitignore?

**Comment by greg0ire** [comment]
@OskarStark can you check PHPStan?

File: .gitignore (hunk 1 above)
Comment (OskarStark) [reply .gitignore]: Hmm yes I think we could manage them mostly the same. Let's see afterwards if we have some special cases and add them via an `{%if ...}` like for other config files, WDYT? Open to create a PR?

**Comment by OskarStark** [comment]
> @OskarStark can you check PHPStan?

I fixed the image itself, these are real errors reported by PHPStan. cc @core23 are you willing to fix them?

File: .gitignore (hunk 1 above)
Comment (OskarStark) [reply .gitignore]: Will do it

**Comment by greg0ire** [comment]
> these are real errors

To me it looks like composer deps are never installed and that's what causing the errors…

**Comment by OskarStark** [comment]
The action is running:
Screenshot 2020-02-12 09 12 08

**Comment by greg0ire** [comment]
And it complains about not being able to find classes from vendors

File: .gitignore (hunk 1 above)
Comment (greg0ire) [reply .gitignore]: So we would list other config files in `projects.yaml` or what?

**Comment by greg0ire** [comment]
@OskarStark I have no issue locally with phpstan so I think it really is an issue the workflow (maybe there should be more steps?)

**Comment by greg0ire** [comment]
@OskarStark I think it is because the image uses `composer install --no-dev`: https://github.com/OskarStark/phpstan-ga/blob/8991b5f626c5d1f2f825916d22f6298269c2d9c9/entrypoint.sh#L3

This is incompatible with the concept of optional dependencies we sometimes use in Sonata.

**Comment by OskarStark** [comment]
@silasjoisten can you please have a look?

**Comment by greg0ire** [comment]
Let's merge as is for now, but it would be great to have some solution for this.

**Pull request merged**